### PR TITLE
Ngx permissions

### PR DESCRIPTION
**Pull request by ynotLeft**
Example of permission-based access to routes and html elements

### CODE REVIEW

File: e2e/scoped.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  it('Admin panel should not be accessible for default user', () => {
+    page.navigateTo('/#/en/pages/scoped');
+    expect(page.getTextBySelector('.mat-card-title')).toEqual('404');
+  });
+
+  it('Admin panel should be accessible for admin', () => {
```
Comment (LehaMotovilov) [inline e2e/scoped.e2e-spec.ts]: screen shot 2018-02-11 at 14 20 47

File: package.json [hunk 2]
```diff
@@ -38,6 +38,7 @@
     "localize-router": "^1.0.0-rc.3",
     "localize-router-http-loader": "^1.0.1",
     "lodash": "^4.17.4",
+    "ngx-permissions": "^3.2.2",
```
Comment (LehaMotovilov) [inline package.json]: Commit `package-lock.json` please.

File: e2e/scoped.e2e-spec.ts (hunk 1 above)
Comment (ynotLeft) [reply e2e/scoped.e2e-spec.ts]: should be fine now

File: package.json (hunk 2 above)
Comment (ynotLeft) [reply package.json]: done

**Comment by ynotLeft** [comment]
also changed markup aka 'verstka' (c) your HTML guy

**Pull request merged**